code/TestAutomation.Tests.BDD/Steps/ElementSteps/CareersButtonSteps.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using TestAutomation.Epam.PageObjects.Pages;
+
+namespace TestAutomation.Tests.BDD.Steps.ElementSteps
+{
+    [Binding]
+    public class CareersButtonSteps : BaseStep
+    {
+        protected MainPage MainPage { get; set; }
+
+        public CareersButtonSteps()
+        {
+            MainPage = new MainPage(Driver);
+        }
+
+        [When(@"I hover over the '([^""]*)' click on Join our team link from the dropdown menu")]
+        public void WhenIHoverOverTheClickOnJoinOurTeamLinkFromTheDropdownMenu(string careers)
+        {
+            MainPage.NavigateToJobListingsPage();
```

Comment:
It is a great practice! calling methods instead of webelements make code extremely readable and nice!

Reply:
thanks